brave/brave-ios
# Fix #1416: Add user onboarding.

## Submitter Checklist:



## Test Plan:


- Choose a search engine, verify it is set for normal and private browsing search.

- Set region to Germany/France, verify a different search engine is set as default in search engine picker in user onboarding(Qwant/Duckduckgo)

### Screenshots:


<img width="860" alt="Zrzut ekranu 2019-08-20 o 11 53 24" src="https://user-images.githubusercontent.com/6950387/63337359-26a2ba00-c341-11e9-9b76-bed6855ebd88.png">

<img width="980" alt="Zrzut ekranu 2019-08-20 o 11 55 47" src="https://user-images.githubusercontent.com/6950387/63337562-7a150800-c341-11e9-92d9-9f1af1a3d752.png">


## Reviewer Checklist:
